examples/server/README.md

```diff
@@ -20,7 +20,7 @@ Command line options:
 - `--lora FNAME`: Apply a LoRA (Low-Rank Adaptation) adapter to the model (implies --no-mmap). This allows you to adapt the pretrained model to specific tasks or domains.
 - `--lora-base FNAME`: Optional model to use as a base for the layers modified by the LoRA adapter. This flag is used in conjunction with the `--lora` flag, and specifies the base model for the adaptation.
 - `-to N`, `--timeout N`: Server read/write timeout in seconds. Default `600`.
-- `--host`: Set the hostname or ip address to listen. Default `127.0.0.1`.
+- `--host`: Set the hostname or ip address to listen. Default `127.0.0.1`. Use `::` for any IPv4+IPv6.
```

Comment:
I think this addition would need to provide at least a little more context.

- 127.0.0.1 is the IPv4 localhost address, 0.0.0.0 is the IPv4 wildcard address.
- ::1 is the IPv6 localhost address.
- 'localhost' should resolve to both IPv4 and IPv6 addresses, on a properly configured system.
- :: is the IPv6 wildcard address, and its meaning depends on the OS - on Windows it binds to IPv6 only. On Linux it will also bind to IPv4 if `sysctl net.ipv6.bindv6only` is zero, which is usually the case. I'm not sure about macOS.

I think there would need to be a code change to reliably listen on both stacks on any OS.

Reply:
What change? Try it. It works on both stacks already.